source/content/guides/drupal/drupal-hosted-btworkflow/01-introduction.md

```diff
@@ -1,38 +1,38 @@
 ---
-title: Migrate a Site Created With the Pantheon Dashboard to Drupal 9 + Build Tools
+title: Migrate a Site Created With the Pantheon Dashboard to Drupal + Build Tools
 subtitle: Introduction
-description: Learn how to migrate a Drupal 8 Site to Drupal 9
-cms: "Drupal 9"
+description: Learn how to migrate a Drupal 8 Site to the latest version of Drupal
```

Comment:
@stovak , should it be "Learn how to update a site to the latest version of Drupal" (omitting ref to D8)?